automation/repeated_test.sh

```diff
@@ -173,12 +188,9 @@ if should_skip_test_run_due_to_too_many_tests "${NEW_TESTS}"; then
     exit 0
 fi
 
-# for some tests we need three nodes aka two nodes with cpu manager installed, thus we grep whether the skip is present
-KUBEVIRT_NUM_NODES=2
-# shellcheck disable=SC2046
-if grep -q 'RequiresTwoWorkerNodesWithCPUManager' $(echo "${NEW_TESTS}" | tr '|' ' '); then
-    KUBEVIRT_NUM_NODES=3
-fi
+# for some tests we need three nodes aka two nodes with cpu manager installed
+# TODO: check whether no test with label `RequiresTwoWorkerNodesWithCPUManager` is present, in that case use two nodes
```

Comment:
Shouldn't we tackle that TODO here? Otherwise the lane will use 1.5 times the resources it needs most of the time!

Reply:
apart from that the resource request on the cluster will not change

Reply:
Great idea - I'll take a look at the hack!

Reply:
Hm, looking at my hack I find it pretty ugly - I've been talking to onsi, the Ginkgo maintainer, and he was suggesting to leverage the dry-run output to improve the properties extraction for the tests. [1]

So I plan to do a followup where I will improve the property extraction in the way he suggested. This will also benefit on what @fossedihelm is wanting to work on (topic is labelling PRs if tests with a certain decorator are changed). I think this would benefit both areas.

@jean-edouard WDYT - are you ok with doing this as a followup? I've created https://github.com/kubevirt/project-infra/issues/4029 for that matter.

[1]: https://github.com/onsi/ginkgo/issues/1521#issuecomment-2701505559